source/includes/plugin_info/_000-describe.md.erb

```diff
@@ -1,11 +1,80 @@
+      name: "GitHub Pull Requests Builder",
+      version: "1.3.3",
+      target_go_version: "15.1.0",
+      description: "Plugin that polls a GitHub repository for pull requests and triggers a build for each of them",
+      target_operating_systems: []
+    }
+%>
+
+<%=
+describe_sub_object 'The plugin about object', json: about_json, html_id: 'plugin-about' do
+  name    'String', 'Name of the plugin'
+  version 'String', 'Version of the plugin that\'s installed'
+  target_go_version 'String', 'The version of GoCD from which the plugin is compatible with GoCD.'
+  description 'String', "Short description about what the plugin does"
+  target_operating_systems 'Array', 'List of operating systems on which the plugin is expected to work.'
+end
+%>
+
+<%=
+describe_sub_object 'The extension_info object', html_id: 'extension-info' do
+"The extension info can contain one or more of the following depending on the plugin extension."
+
+
+  plugin_settings         'Object', 'List of [settings](#pluggable-instance-settings) required to configure a plugin if the plugin has any settings that could be configured.'
+  profile_settings        'Object', 'List of [settings](#pluggable-instance-settings) required to configure an elastic profile. This is shown for a plugin implementing the [elastic-agent](https://plugin-api.gocd.org/current/elastic-agents/) extension.'
+  auth_config_settings    'Object', 'List of [settings](#pluggable-instance-settings) required to configure the auth config object. This is shown for a plugin implementing the [authorization](https://plugin-api.gocd.org/current/authorization/) extension.'
+  role_config_settings    'Object', 'List of [settings](#pluggable-instance-settings) required to configure role config. This is shown for a plugin implementing the [authorization](https://plugin-api.gocd.org/current/authorization/) extension.'
+  scm_settings            'Object', 'List of [settings](#pluggable-instance-settings) required to configure a pluggable scm instance. This is shown for a plugin implementing the scm extension.'
+  task_settings           'Object', 'List of [settings](#pluggable-instance-settings) required to configure a pluggable task instance. This is shown for a plugin implementing the [task](https://plugin-api.gocd.org/current/tasks/) extension.'
+  package_settings        'Object', 'List of [settings](#pluggable-instance-settings) required to configure a package in a repository. This is shown for a plugin implementing the package-repository extension.'
+  repository_settings     'Object', 'List of [settings](#pluggable-instance-settings) required to configure a package repository plugin. This is shown for a plugin implementing the package-repository extension.'
+  capabilities            'Object', '[Key value pairs](#capabilities) which informs the user whether a plugin supports a particular capability. Currently, this information is shown for some authorization and elastic agent plugins.'
+end
```

Comment:
Do you want to consider grouping the extension_info by plugin, it would make it easier to know the plugin specific settings?